specification/index.bs

```diff
@@ -149,6 +149,10 @@ This key may be present.
 
 This key may be present.
 
+### Key `trial_tokens`
```

Comment:
There is another section in the doc that enumerates all keys, under the following heading:

> The following keys must be considered valid:

Shouldn't we also add this new key there?